### PR TITLE
Update module.md and examples according to comments

### PR DESCRIPTION
**Pull request by sahanHe**
# Description
related to - 
Fixes # [4844](https://github.com/ballerina-platform/ballerina-library/issues/4844)

Related Pull Requests (remove if not relevant)
- Pull request 1
- Pull request 2

One line release note: 
- One line describing the feature/improvement/fix made by this PR 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Ballerina Version:
* Operating System:
* Java SDK: 

# Checklist:

### Security checks
 - [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: examples/ftp-edi-message-to-salesforce-opportunity/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[package]
+org = "integration_samples"
```
Comment (NipunaRanasinghe) [inline examples/ftp-edi-message-to-salesforce-opportunity/Ballerina.toml:2]:
```suggestion
org = "salesforce_samples"
```

Comment (NipunaRanasinghe) [reply examples/ftp-edi-message-to-salesforce-opportunity/Ballerina.toml:2]: apply where necessary

File: ballerina/Package.md [hunk 2]
```diff
@@ -12,8 +12,9 @@ The `ballerinax/salesforce` is a [Ballerina](https://ballerina.io/) connector fo
 |                     | Version            |
 |---------------------|--------------------|
 | Ballerina Language  | Swan Lake 2201.8.0 |
-| Salesforce REST API | v48.0              |
-| Salesforce Bulk API | v1                 |
+| Salesforce REST API | v59.0              |
```
Comment (NipunaRanasinghe) [inline ballerina/Package.md:15]: As we don't use Package.md in central anymore, Package.md should duplicate the exact Module.md content

**Review by NipunaRanasinghe — CHANGES_REQUESTED** [review]

File: ballerina/Package.md (hunk 2 above)
Comment (sahanHe) [reply ballerina/Package.md]: updated in [5452761](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/319/commits/54527610e82929dda4ebc8f90016a66c39eb1501)

File: examples/ftp-edi-message-to-salesforce-opportunity/Ballerina.toml (hunk 1 above)
Comment (sahanHe) [reply examples/ftp-edi-message-to-salesforce-opportunity/Ballerina.toml]: updated in [5452761](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/319/commits/54527610e82929dda4ebc8f90016a66c39eb1501)

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Pull request merged**